package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "@vaadin/vaadin-messages",
+  "version": "1.0.0-alpha1",
```

Comment:
Please update this to `0.1.0` as the version needs to be bumped to `1.0.0-alpha1` during the first release.

Reply:
Done.